sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -611,6 +647,17 @@ scatter_rgba(T *p, simd<uint32_t, N> offsets,
       addrs.data(), vals.data(), mask.data());
 }
 
+template <rgba_channel_mask RGBAMask = rgba_channel_mask::ABGR, typename T,
```

Comment:
Here and in other new overloads:
please add doxygen, and explain what is the difference with the other `scatter_rgba` overload.